device.go

```diff
@@ -2,6 +2,7 @@ package gbridge
 
 import "github.com/pborges/gbridge/proto"
 
+// Device represents a Smart Home Device with an Name, ID and Function
```

Comment:
... a Smart Home Device with a Name, ID, and other metadata as defined by google <insert link here>

Reply:
Also probably mention the list of other interfaces to add metadata to Device i.e ```DeviceInfoProvider```, ```DeviceRoomHintProvider``` etc